### DISCUSSION TITLE
考虑PCL为多语言版本的预设主页添加切换方式吗（

### DISCUSSION THREAD

**Discussion by Joker2184** [Meloong-Git/PCL#5080] [提问] [answered]
![QQ截图20241109111131](https://github.com/user-attachments/assets/ae46ba74-b96f-40a4-8ed7-5d2bb839dc21)
（abb

**Comment by tangge233** [comment, marked as answer]
### 正在考虑与设计，请坐和放宽.jpg

编辑：https://github.com/Hex-Dragon/PCL2/pull/4145#issuecomment-2466060602

**Comment by MoYuan-CN** [comment]
你的图片并没有做出多语言适配…… 这个东西要适配的话需要做很多额外工作……

**Reply by Joker2184** [reply]
龙猫的图....
问题不大
能知道在干啥就行

**Comment by wuliaodexiaoluo** [comment]
如果这样的话...
是不是应该额外加个 Accept-Language?
